legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/tests/relationalSetUp.pure

```diff
@@ -40,7 +40,7 @@ Database meta::relational::tests::dbInc
        firstName: personTable.FIRSTNAME  PRIMARY KEY,
        address :  @Address_Person | addressTable.NAME PRIMARY KEY
     )
-   
+
```

Comment:
I had to introduce ID in mapping to be able to sort expected output. Otherwise we could not guarantee integrity of tests when comparing arrays